contrib/lncli.bash-completion

```diff
@@ -17,7 +17,7 @@ _lncli() {
     case "$prev" in
         # example of further completion
         newaddress)
-            COMPREPLY=( $( compgen -W "p2wkh np2wkh p2pkh" -- "$cur" ) )
+            COMPREPLY=( $( compgen -W "p2wkh np2wkh" -- "$cur" ) )
```

Comment:
Could you please clarify the commit message a bit. Something like: "remove bash completion for unsupported address type p2pkh2